api/src/main/java/org/openmrs/module/fhir2/api/translators/ServiceRequestTranslator.java

```diff
@@ -12,7 +12,7 @@
 import org.hl7.fhir.r4.model.ServiceRequest;
 import org.openmrs.TestOrder;
 
-public interface ServiceRequestTranslator extends OpenmrsFhirUpdatableTranslator<TestOrder, ServiceRequest> {
+public interface ServiceRequestTranslator<T> extends ToFhirTranslator<T, ServiceRequest> {
 	
```

Comment:
This is a one-way translation. Don't need to translate back to openmrs type? Or you are planning to add it later?
   
```
 public interface ServiceRequestTranslator<T> extends ToFhirTranslator<T, ServiceRequest>, 
         ToOpenmrsTranslator<T, ServiceRequest>{
```

Reply:
Eventually yes, but for our present purposes, no. Basically this part just removed some `return null;` implementations.